packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -49,6 +49,13 @@ export namespace ProblemsCommands {
     export const COPY_MESSAGE: Command = {
         id: 'problems.copy.message',
     };
+    export const CLEAR_ALL: Command = {
+        id: 'problems.clear.all'
```

Comment:
A `label` is necessary in order to be displayed in the command palette.
At the moment this command is not available at all.